test/feature/dynamic_types/dds_types_tests/DynamicTypesArraysDDSTypesTests.cpp

```diff
@@ -588,6 +590,65 @@ TEST_F(DynamicTypesDDSTypesTest, DDSTypesTest_ArrayOctet)
     EXPECT_EQ(DynamicDataFactory::get_instance()->delete_data(data), RETCODE_OK);
 }
 
+void DDSTypesTest_ArrayUInt8_common()
+{
+    eprosima::fastdds::dds::DynamicTypesDDSTypesTest_DDSTypesTest_ArrayOctet_Test support;
```

Comment:
No need to instantiate the test, you could pass it by reference from the tests:

```suggestion
void DDSTypesTest_ArrayUInt8_common(
        eprosima::fastdds::dds::DynamicTypesDDSTypesTest& support)
{
```